student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(name: 'Unknown', age: nil, parent_permission: true, classroom: 'No Classroom')
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Once again as mentioned previously the `age` variable in the constructor above shouldn't have a default value of `nil` as it should be specified whenever an instance of a student class is created so please make sure to remove it.

Reply:
Kindly I didn't understand this one...